src/pages/settings/company/create/CompanyCreate.tsx

```diff
@@ -0,0 +1,77 @@
+/**
+ * Invoice Ninja (https://invoiceninja.com).
+ *
+ * @link https://github.com/invoiceninja/invoiceninja source repository
+ *
+ * @copyright Copyright (c) 2022. Invoice Ninja LLC (https://invoiceninja.com)
+ *
+ * @license https://www.elastic.co/licensing/elastic-license
+ */
+
+import { Button } from '@invoiceninja/forms';
+import { AxiosError } from 'axios';
+import { endpoint } from 'common/helpers';
+import { request } from 'common/helpers/request';
+import { route } from 'common/helpers/route';
+import { toast } from 'common/helpers/toast/toast';
+import { Modal } from 'components/Modal';
+import { useState, SetStateAction, Dispatch } from 'react';
+import { useTranslation } from 'react-i18next';
+
+interface Props {
+  isModalOpen: boolean;
+  setIsModalOpen: Dispatch<SetStateAction<boolean>>;
+}
+
+export function CompanyCreate(props: Props) {
+  const [t] = useTranslation();
+
+  const [isFormBusy, setIsFormBusy] = useState<boolean>(false);
+
+  const handleSave = async () => {
+    if (!isFormBusy) {
+      toast.processing();
+
+      setIsFormBusy(true);
+
+      try {
+        await request('POST', endpoint('/api/v1/companies'));
+
+        toast.success(t('created_company'));
```

Comment:
`toast` itself is translatable. No need to pass a translated string.

```ts
toast.success('created_company');
```

Reply:
> `toast` itself is translatable. No need to pass a translated string.
> 
> ```ts
> toast.success('created_company');
> ```

Great to know. I will keep in mind for for other features.